.github/workflows/benchmark-wildcat.yml

```diff
@@ -0,0 +1,116 @@
+name: Benchmark GTSFM on select datasets on the wildcat machine
+
+################################################################################
+# Please only run this action after making sure that the shoter benchmarks
+# (defined in benchmark.yml) all run correctly and show no major regressions.
+#
+# Please be conscious of multiple queued actions, as they run sychronously on
```

Comment:
why do we have these instructions here if the workflow dispatches automatically on pull requests?